README.md

```diff
@@ -605,17 +652,6 @@ apollo {
 }
 ```
 
-### Use system pre-installed `apollo-codegen`
```

Comment:
Should we really remove this? I think it is still helpful until npm is completely removed

Reply:
The reasonning was that the README.md is for new users and we want to onboard them on the new CodeGen but I don't mind keeping that a bit longuer. I'll revert.